source/_components/hive.markdown

```diff
@@ -19,18 +20,22 @@ redirect_from:
   - /components/switch.hive/
 ---
 
-The `hive` integration is the main integration to set up and integrate all supported Hive devices. Once configured with the minimum required details it will detect and add all your Hive devices into Home Assistant, including support for multizone heating.
+The `hive` integration is the main integration to set up and integrate all supported Hive devices. Once configured with the minimum required details it will detect and add all Hive devices into Home Assistant, including support for multi-zone heating.
 
 This integration uses the unofficial API used in the official Hive website [https://my.hivehome.com](https://my.hivehome.com), and you will need to use the same Username and Password you use on the Hive website to configure this Hive integration in Home Assistant.
 
 There is currently support for the following device types within Home Assistant:
 
-- [Binary Sensor](#Binary-Sensor)
-- [Climate](#Climate)
-- [Light](#Light)
-- [Sensor](#Sensor)
-- [Switch](#Switch)
-- [Water Heater](#Water-Heater)
+- [Services](#services)
```

Comment:
Service is not a device type 😉

Reply:
I have updated the description to advise these are the services and platforms supported in home assistant.